src/NUnitConsole/nunit-console/Utilities/TeamCityEventHandler.cs

```diff
@@ -21,7 +21,6 @@
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 // ***********************************************************************
 
-using System;
```

Comment:
Redundant code is just that redundant.  Add it back if you need it.

On Saturday, April 11, 2015, CharliePoole <notifications@github.com> wrote:

> In src/NUnitConsole/nunit-console/Utilities/TeamCityEventHandler.cs
> <https://github.com/nunit/nunit/pull/586#discussion_r28157334>:
>
> > @@ -21,7 +21,6 @@
> >  // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
> >  // ***********************************************************************
> >
> > -using System;
>
> I can go either way on this. Often, I exempt System from the requirement
> that all using statements be needed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/586/files#r28157334>.
>